### PR TITLE
(antu) add basic services excluding observability

### PR DESCRIPTION
**Pull request by jhoblitt**
Related to https://github.com/lsst-it/lsst-control/pull/1201

### CODE REVIEW

File: antu/rke/cluster.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+services:
+  kubelet:
+    extra_args:
+      node-status-max-images: "-1"
+      max-pods: 250
+network:
+  plugin: canal
+ssh_key_path: ~/.ssh/id_rsa
+ignore_docker_version: true
+kubernetes_version: v1.27.12-rancher1-1
```
Comment (cbarria) [inline antu/rke/cluster.yml:48]: can this cluster start at 1.28.9 please 🙏🏻

**Review by cbarria — CHANGES_REQUESTED** [review]
lets start with k8s version 1.28.9, the rest looks great 👍🏻

File: antu/rke/cluster.yml (hunk 1 above)
Comment (jhoblitt) [reply antu/rke/cluster.yml]: Done.   Please don't forget to remove the rke version string from the antu cluster when updating `ls`.

**Review by cbarria — APPROVED** [review]
Looks Great 👍🏻

**Pull request merged**